src/storage/v2/durability/snapshot.cpp

```diff
@@ -225,20 +226,27 @@ void LoadPartialEdges(const std::filesystem::path &path, utils::SkipList<Edge> &
   if (!snapshot.SetPosition(from_offset)) throw RecoveryFailure("Couldn't read data from snapshot!");
 
   std::vector<std::pair<PropertyId, PropertyValue>> read_properties;
+  uint64_t five_percent_chunk = edges_count / 20;
```

Comment:
What about when `edges_count < 20`? `five_percent_chunk == 0`

info log will look strange as not all the 5% messages will exist. Maybe a cause for confusion/unnecessary support queries.

debug log will look strange as it will be "egde set <0 - 0>", "edge set <1 - 1>" etc. Not a problem purely cosmetic.

Reply:
we'll keep it as is for now

Reply:
This has been included in the latest commit.